custom_metrics/css.js

```diff
@@ -0,0 +1,30 @@
+//[css]
+// Uncomment the previous line for testing on webpagetest.org
+
+return JSON.stringify({
+  'css_in_js': (() => {
+    const CssInJsMap = {
+      'styled-components': !!document.querySelector('style[data-styled],style[data-styled-components]'),
```

Comment:
Rather than IDs/keys like `'styled-components'` could we use the more "human readable" names like `'Styled Components'` to avoid the need for converting later in the pipeline?

Reply:
Started doing that and "Merge Styles" looks a bit too generic. Now, I'm thinking maybe we should use npm package names? For example "@uifabric/merge-styles" and so on?

Reply:
Let's opt for maximal readability since these values will be shown to readers in data visualizations. I think even a generic name like "Merge Styles" is preferable to its npm package name.

Reply:
Sounds good, updated